Code/Framework/AzToolsFramework/Platform/Linux/AzToolsFramework/API/PythonLoader_Linux.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/API/EditorPythonConsoleBus.h>
+#include <dlfcn.h>
+
+namespace AzToolsFramework::EmbeddedPython
+{
+    class PythonLoader
```

Comment:
If you move the declaration of this class to `PythonLoader.h`, then it can be re-used for the Default and Linux implementations

```
class PythonLoader final
{
    PythonLoader()
    ~PythonLoader()
 private:
    void* m_embeddedLibPythonHandle{ nullptr };
};
```